PostMetabox.php

```diff
@@ -292,7 +292,8 @@ protected function save( $post_id ) {
 			);
 			update_post_meta( $post_id, '_error_data_' . $this->id, $error_data );
 
-			$location = add_query_arg( 'message', 1, get_edit_post_link( $post_id, 'url' ) );
+			$location = esc_url_raw( add_query_arg( 'message', 1, get_edit_post_link( $post_id, 'url' ) ) );
+
 			wp_redirect( apply_filters( 'redirect_post_location', $location, $post_id ) );
```

Comment:
My argument boils down to this:

It's better to escape data after passing it through a filter, so that any callback attached to that filter doesn't have to also escape it.

If you have a counter argument to this specific point, I'd be glad to hear it. If not, please make the requested change.